### PR TITLE
Add Korok IDs as column in objs table

### PR DESCRIPTION
**Pull request by savage13**
Using the same file `korok_ids.json` that is in the objmap repository, add the Korok IDs to the objs table.  

- Added column `korok`
- Searching on Korok IDs is available using `korok: `
- Korok IDs are returned as `korok` in the JSON response

I was not sure where to put `korok_ids.json`, so it is in the root directory (and in the objmap repo as well).

---
This change is [Reviewable](https://reviewable.io/reviews/zeldamods/radar/11)

### CODE REVIEW

File: build.ts [hunk 1]
```diff
@@ -191,9 +196,9 @@ db.exec(`
 
 
 const insertObj = db.prepare(`INSERT INTO objs
-  (map_type, map_name, map_static, gen_group, hash_id, unit_config_name, ui_name, data, one_hit_mode, last_boss_mode, hard_mode, disable_rankup_for_hard_mode, scale, sharp_weapon_judge_type, 'drop', equip, ui_drop, ui_equip, messageid, region, field_area, spawns_with_lotm)
+  (map_type, map_name, map_static, gen_group, hash_id, unit_config_name, ui_name, data, one_hit_mode, last_boss_mode, hard_mode, disable_rankup_for_hard_mode, scale, sharp_weapon_judge_type, 'drop', equip, ui_drop, ui_equip, messageid, region, field_area, spawns_with_lotm, korok)
```
Comment (leoetlino) [inline build.ts:199]: Can we call this korok_id instead? The FTS column can keep its current name but I think the column name for the objs table should be more precise.

Comment (savage13) [reply build.ts:199]: Yep

File: app/app.ts [hunk 2]
```diff
@@ -42,10 +42,11 @@ function parseResult(result: any): { [key: string]: any } {
   // Most objects do not have DisableRankUpForMasterMode set, so don't include it unless it is set.
   result.disable_rankup_for_hard_mode = result.disable_rankup_for_hard_mode ? true : undefined;
   result.pos = [Math.round(result.data.Translate[0] * 100) / 100, Math.round(result.data.Translate[2] * 100) / 100];
+  result.korok = result.korok || undefined;
   return result;
 }
 
-const FIELDS = 'objid, map_type, map_name, map_static, hash_id, unit_config_name as name, `drop`, equip, data, messageid, scale, sharp_weapon_judge_type, hard_mode, disable_rankup_for_hard_mode, spawns_with_lotm, field_area';
+const FIELDS = 'objid, map_type, map_name, map_static, hash_id, unit_config_name as name, `drop`, equip, data, messageid, scale, sharp_weapon_judge_type, hard_mode, disable_rankup_for_hard_mode, spawns_with_lotm, field_area, korok_id as korok';
```
Comment (leoetlino) [inline app/app.ts:49]: Do we need the `as korok` here? If it's for the frontend I think it should just use "korok_id" instead of "korok".

**Comment by savage13** [comment]
Changed `korok` column to `korok_id`

Made sure if `korok` was undefined it was not returned with the JSON result.

File: app/app.ts (hunk 2 above)
Comment (savage13) [reply app/app.ts:49]: The FTS search term is currently `korok:` but  returns a `korok_id`.  If this is ok, I will make the change.

It is ok with me.

Comment (leoetlino) [reply app/app.ts:49]: Yeah that sounds fine to me.

**Review by leoetlino — APPROVED** [review]

**Pull request merged**